snuba/web/query.py

```diff
@@ -50,6 +50,16 @@
 metrics = MetricsWrapper(environment.metrics, "api")
 
 MAX_QUERY_SIZE_BYTES = 256 * 1024  # 256 KiB by default
+EXCLUDED_REFERRERS_FROM_FIELDS_SORTING = {
```

Comment:
How did you determine this list?

Reply:
This list was chosen based on the highest average queries by referrers to our cache. Removing them (along with using a variable percentage of queries to roll-out this experiment, it will help ensure our cache does not become overloaded. Do you have suggestions/more context to what should be removed or added?

Reply:
I would also add `group.unhandled-flag`. If we think of any others we can add them later.